config/schema/modules/aws-s3.yaml

```diff
@@ -3,6 +3,12 @@ name: str(required=False)
 env_name: str(required=False)
 layer_name: str(required=False)
 module_name: str(required=False)
-bucket_name: str(required=False)
+bucket_name: str(required=True)
 block_public: any(required=False)
-bucket_policy: any(required=False)
\ No newline at end of file
+bucket_policy: any(required=False)
+cors_rule: 
```

Comment:
is this required=False by default?

Reply:
Ah it's not! Let me figure out a way to fix it